composer.json

```diff
@@ -56,7 +56,8 @@
         "studio-42/elfinder": "2.1.61",
         "symfony/translation": "^3.4",
         "tedivm/stash": "~0.16",
-        "vlucas/phpdotenv": "^5.4"
+        "vlucas/phpdotenv": "^5.4",
+        "league/event": "^3.0"
```

Comment:
Sort package alphabetically. Same for `packages/app/composer.json`.

Reply:
👍